Makefile.PL

```diff
@@ -360,6 +360,9 @@ if (-d 'inc/patches') {
         for my $file (glob 'inc/patches/*.patch') {
             # replacing \n with \n will change \n to \r\n on Win, without this patch.exe will crash
             system "cd \Q$mxbase\E && perl -pe \"s/\\n/\\n/\" ../../\Q$file\E | patch -p1 -N";
+
+            # some Windows systems don't like the path escape ...
+            # system "cd $mxbase && perl -pe \"s/\\n/\\n/\" ../../$file | patch -p1 -N";
```

Comment:
Windows 7, Perl 5.12.4, Visual Studio 10. It's just the standard command line. I put it there just because I'm getting tired to switch the escapes on/off. Now it's just moving the # 3 lines up.